pages/components/navigation/navigation.css

```diff
@@ -43,7 +57,44 @@
 .bd_num {
   left: 1.5rem;
 }
-.sui_search .sui_input input {
+.sui_search .sui_input input,
+.sui_search_mobile .sui_input input {
   padding: 6px 15px 4px 15px;
   padding-left: 3.6rem;
 }
+.fa-bars {
+  font-size: 2.6rem;
+  display: none;
+}
+@media only screen and (max-width: 768px) {
+  .sui_nav_mobile {
+    gap: 2rem;
+  }
+  .sui_nav_mobile .sui_search {
+    display: none;
+  }
+  .sui_search_mobile {
+    display: revert;
+  }
+}
+@media only screen and (max-width: 600px) {
+  .sui_cta_wrapper {
+    flex-direction: column;
+  }
+  .sui_nav_mobile .sui_cta {
+    flex-direction: column;
+    position: absolute;
+    background: var(--BG);
+    top: 5rem;
+    right: -1rem;
+    z-index: 9;
+    padding: 2rem;
```

Comment:
Why are you using high value of z-index ?

Reply:
To override the existing value of my Prism Highlight.